app/src/main/res/drawable/ic_chevron_down.xml

```diff
@@ -7,6 +7,6 @@
         android:height="24dp"
         android:viewportWidth="24"
         android:viewportHeight="24">
-    <path android:fillColor="?primaryText"
+    <path android:fillColor="@color/primary_text_normal_theme"
```

Comment:
WIll test this tomorrow to see what the actual changes look like with the different themes, but this change will mean we won't show the PRIVATE primary text color when ic_chevron_down.xml is used in private mode.

Reply:
Sure, please let me know what you think after testing.  Currently when we're in private mode, the text for both inactive tabs and group tabs are not readable.  Thanks